test/page-detect.js

```diff
@@ -192,6 +192,15 @@ test('isCompare', urlMatcherMacro, pageDetect.isCompare, [
 	'https://github.com/sindresorhus/refined-github/graphs'
 ]);
 
+test('isQuickPR', urlMatcherMacro, pageDetect.isQuickPR, [
+	'https://github.com/sindresorhus/refined-github/compare/master...branch-name?quick_pull=1',
+	'https://github.com/sindresorhus/refined-github/compare/branch-1...branch-2?quick_pull=1'
```

Comment:
Could you please add positive tests with a single branch (where the default base branch is implied)?